composer.json

```diff
@@ -28,7 +28,7 @@
 		"libphutil": "1.2"
 	},
 	"require-dev": {
-		"behat/behat": "~3.0",
+		"behat/behat": "v3.1.0rc1",
```

Comment:
Agreed. I get
```
  Problem 1
    - Installation request for behat/behat ~3.1 -> satisfiable by behat/behat[v3.1.0rc1].
    - Removal request for behat/behat == 3.1.0.0-RC1 ```

when setting it to `~3.1`. `>=3.1.0rc1` works though. I'm not sure if I'm using composer wrong. Any ideas?